app/src/main/java/upday/droidconmvvm/MainActivity.java

```diff
@@ -11,16 +11,14 @@
 
 import java.util.List;
 
-import rx.android.schedulers.AndroidSchedulers;
-import rx.schedulers.Schedulers;
-import rx.subscriptions.CompositeSubscription;
-import upday.droidconmvvm.datamodel.IDataModel;
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.schedulers.Schedulers;
+import io.reactivex.disposables.CompositeDisposable;
 import upday.droidconmvvm.model.Language;
 
 public class MainActivity extends AppCompatActivity {
 
-    @NonNull
-    private CompositeSubscription mSubscription;
+    @NonNull CompositeDisposable compositeDisposable;
```

Comment:
make this private, keep the `@NonNull` on a different line, name this `mCompositeDisposable`

Reply:
done